controllers/front/payment.php

```diff
@@ -343,6 +343,17 @@ private function displayError($blockonomics)
                 'Please use Test Setup button in configuration to diagnose the error ',
                 'payment'
             );
+        
+        if (isset($responseObj) && isset($responseObj->data) && isset($responseObj->data->message)) {
+            $error_message = $responseObj->data->message;
+            $lowercase_error_message = strtolower($error_message);
+
+            if (strlen($error_message) > 0 && (strpos($lowercase_error_message, 'gap limit') !== false || strpos($lowercase_error_message, 'temporary') !== false)) {
+                $unable_to_generate = $unable_to_generate . '<p>
+                    <strong>More Info:</strong> ' . $responseObj->data->message . 
```

Comment:
The _More Info_ and _Test setup_ message are not required and should look as follows:
![Wordpress](https://user-images.githubusercontent.com/39666372/141820423-6ba0e202-2b26-4583-a390-09529f591bbc.png)
https://github.com/blockonomics/woocommerce-plugin/blob/master/templates/blockonomics_checkout.php#L37-L41
```

Reply:
Okay

Reply:
@DarrenWestwood After replacing the generic message with the message from API, this is the message I'm getting. This seems different from the one received in Woocommerce. while both of them are showing error from the API Only.

![image](https://user-images.githubusercontent.com/22245433/144179452-3e01eb78-c83e-4de0-9f25-b1feac175570.png)